docs/security/apparmor.md

```diff
@@ -97,6 +97,133 @@ $ apparmor_parser -R /path/to/profile
 $ /etc/init.d/apparmor start
 ```
 
+### Resources for writing profiles
+
+The syntax for file globbing in AppArmor is a bit different than some other
+globbing implementations. It is highly suggested you take a look at some of the
+below resources with regard to AppArmor profile syntax.
+
+- [Quick Profile Language](http://wiki.apparmor.net/index.php/QuickProfileLanguage)
+- [Globbing Syntax](http://wiki.apparmor.net/index.php/AppArmor_Core_Policy_Reference#AppArmor_globbing_syntax)
+
+## Examples
+
+### Nginx Custom Profile
+
+Let's create a custom AppArmor profile for nginx to show how this works.
+
+1. Save the custom profile to disk.
+
+Below is our custom profile we will save in `/etc/apparmor.d/containers/docker-nginx`.
+
+> **NOTE:** The path we save to is not a requirement to be the same as above,
+> this was only used as the path for this example.
+
+```
```

Comment:
this block is not right, as number list would be not correctly display. You may need more space here to make number list can correctly show